compiler/testData/cli/jvm/firHello.args

```diff
@@ -1,3 +1,4 @@
+// IGNORE_BACKEND: ANY
```

Comment:
Maybe comment out the `kotlin.io.println` call instead, since the rest of the test works?

cc @demiurg906

Reply:
You're right, that makes more sense! Done.